job-templates/kubernetes_release_staging.json

```diff
@@ -77,7 +77,7 @@
       "windowsPublisher": "MicrosoftWindowsServer",
       "windowsOffer": "WindowsServer",
       "windowsSku": "2019-Datacenter-Core-with-Containers-smalldisk",
-      "imageVersion": "17763.1098.2003062342"
```

Comment:
Is this kubernetes_release_staging.json used anywhere?
I looks like it is still pointing to 1.17...

Reply:
Yes it's used in `pull-kubernetes-e2e-aks-engine-azure-windows`: https://github.com/kubernetes/test-infra/blob/dd6a466605560e9cbe9a4a2975673cf61dfc7c59/config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml#L55

If I understand the test-grid config correctly it builds the kubebinaries from the PR?  @chewong is that correct?

Reply:
Yes that's correct

Reply:
ah, that is a bit confusing but ok